src/components/RecommendDetail.js

```diff
@@ -49,7 +49,7 @@ class RecommendDetail extends Component {
 
           <Button
             accessibilityLabel={'Click to log in!'}
-            onPress={() => Actions.videoViewer({ videoId: this.state.videoId })}
+            onPress={() => Actions.VideoViewer({ videoId: this.state.videoId })}
```

Comment:
No arrow functions as props. Please extract to its own variable.

Reply:
This code was pulled from the master, I didn't touch it with this PR. How should this be resolved?

Reply:
Also, I originally added the yarn and video dependency changes because I didn't want to push code that didn't run.